backend-go/.golangci.yml

```
@@ -0,0 +1,23 @@
+    - revive
+    - goprintffuncname
+    - gosec
+    - lll
+    - misspell
+    - nakedret
+    - wrapcheck
+  disable:
+    - gosimple
+    - staticcheck
+    - structcheck
+    - unused
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)